opensrp-unicef-tunisia/src/main/java/org/smartregister/uniceftunisia/presenter/NavigationPresenter.java

```diff
@@ -38,42 +40,17 @@ public NavigationPresenter(NavigationContract.View view) {
     }
 
     private void initialize() {
-        tableMap.put(GizConstants.DrawerMenu.CHILD_CLIENTS, GizConstants.RegisterType.CHILD);
-        tableMap.put(GizConstants.DrawerMenu.ANC_CLIENTS, GizConstants.RegisterType.ANC);
-        tableMap.put(GizConstants.DrawerMenu.ANC, GizConstants.RegisterType.ANC);
-        tableMap.put(GizConstants.DrawerMenu.ALL_CLIENTS, GizConstants.RegisterType.OPD);
+        tableMap.put(AppConstants.DrawerMenu.CHILD_CLIENTS, AppConstants.RegisterType.CHILD);
+        tableMap.put(AppConstants.DrawerMenu.ANC_CLIENTS, AppConstants.RegisterType.ANC);
+        tableMap.put(AppConstants.DrawerMenu.ANC, AppConstants.RegisterType.ANC);
+        tableMap.put(AppConstants.DrawerMenu.ALL_CLIENTS, AppConstants.RegisterType.OPD);
     }
 
     @Override
     public NavigationContract.View getNavigationView() {
         return mView.get();
     }
 
-    @Override
-    public void refreshNavigationCount() {
-        int navigationItems = 0;
-        while (navigationItems < mModel.getNavigationItems().size()) {
-            final int finalNavigationItems = navigationItems;
-            String menuTitle = mModel.getNavigationItems().get(navigationItems).getMenuTitle();
-            mInteractor.getRegisterCount(tableMap.get(menuTitle),
-                    new NavigationContract.InteractorCallback<Integer>() {
-                        @Override
-                        public void onResult(Integer result) {
-                            mModel.getNavigationItems().get(finalNavigationItems).setRegisterCount(result);
-                            getNavigationView().refreshCount();
-                        }
-
-                        @Override
-                        public void onError(Exception e) {
-                            Timber.e(e, "Error retrieving count for %s",
-                                    tableMap.get(mModel.getNavigationItems().get(finalNavigationItems).getMenuTitle()));
-                        }
-                    });
-            navigationItems++;
-        }
-
```

Comment:
You dont need this? Its responsible for updating the current clients count after registration and sync.

Reply:
We do not have the counts on the navigation drawer so it is not necessary at the moment.